### PR TITLE
Add google season of docs 2023 proposal

### PR DESCRIPTION
**Pull request by steff456**
This PR adds the proposal for Google Season of Docs 2023.

It includes all the sections defined in the template: https://developers.google.com/season-of-docs/docs/org-proposal-template

----
:books: Documentation preview :books:: https://jupyter-accessibility--120.org.readthedocs.build/en/120/

### CODE REVIEW

**Comment by gabalafou** [comment]
Here is the preview URL of the page: https://jupyter-accessibility--120.org.readthedocs.build/en/120/funding/google-season-of-docs-2023.html

**Review by isabela-pf — COMMENTED** [review]
Thanks for sharing this so we can keep track of funding proposals related to Jupyter accessibility efforts! 

I only have one question: would it be helpful to add links to some of the project descriptions and similar areas? This is not a blocker for me to merge it, but I did want to ask because I can add that.

**I'll leave this up for review until Wednesday** so that people have a little more time to review, but the proposal is also up before the deadline. I'll merge it then! Thanks again for leading this!

File: docs/funding/google-season-of-docs-2023.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Proposal Title - organization name
+
+<!---
+Your proposal title should be short and specific. “Update ORGNAME Contributor Guide” is a good proposal title. “Documentation Improvements” is too vague; “Update Sections 5,7,23,99 of Contributor Guide, Create FAQ, and Create Style Guide” is too long.
+-->
+
+Create foundational accessibility documentation for JupyterLab
```
Comment (trallard) [inline docs/funding/google-season-of-docs-2023.md:7]: In the spirit of clarity I suggest:

```suggestion
Create foundational user-focused accessibility documentation for JupyterLab
```

File: docs/funding/google-season-of-docs-2023.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+-->
+
+JupyterLab is an extensible environment for interactive and reproducible computing based on the Jupyter Notebook. It allows scientists, researchers, engineers, journalists, students, and many other users to configure and arrange workflows combining code development, visualization and text inside a single web-based interface. Characterized by its powerful execution engine, Jupyter supports over 40 programming languages, including Python, Julia, R and Scala.
+
+JupyterLab is a user interface offering notebook manipulation, text editor, file browser, and terminal, among other development tools. It has a seamless integration with the most common scientific packages and it is included in multiple downstream applications as a dependency. Moreover, JupyterLab is flexible, powerful and extensible by third-party and official extensions distributed via PyPi, conda and other package managers.
```
Comment (trallard) [inline docs/funding/google-season-of-docs-2023.md:17]:
```suggestion
JupyterLab is a user interface comprising notebook manipulation, text editor, file browser, and terminal, among other development tools. It has a seamless integration with the most common scientific packages and is included in multiple downstream applications as a dependency. Moreover, JupyterLab is flexible, robust, and extensible by third-party and official extensions distributed via PyPi, conda, and other package managers.
```

File: docs/funding/google-season-of-docs-2023.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+# About your project
+
+## Your project’s problem
+
+<!---
+Tell us about the problem your project will help solve. Why is it important to your organization or project to solve this problem?
+-->
+
+The JupyterLab accessibility team has been concentrating its efforts on accessibility remediation through work on interface audits, testing events, improvements to UI/UX and automated testing. Due to the limited number of resources and the prioritization of such remediation work, documenting accessibility features has been a secondary priority. As more accessibility-focused improvements are added, it is becoming increasingly necessary that we provide a dedicated focus on adding user accessibility documentation for JupyterLab.
```
Comment (trallard) [inline docs/funding/google-season-of-docs-2023.md:31]:
```suggestion
The JupyterLab accessibility team has been concentrating its efforts on accessibility remediation through work on interface audits, testing events, improvements to UI/UX and automated testing. Due to the limited resources and the prioritization of such remediation work, documenting accessibility features has been a secondary priority. As more accessibility-focused improvements are added, it is becoming increasingly necessary that we provide a dedicated focus on adding user accessibility documentation for JupyterLab.
```

File: docs/funding/google-season-of-docs-2023.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+Through this project, we aim to create documentation primarily targeted at anyone with a temporary, situational, or permanent disability that wants or needs to use JupyterLab. We want to provide the JupyterLab community with a comprehensive guide about the accessibility features present in the application. For example, developing documentation that answers the questions “What can and can’t I do in JupyterLab with accessibility accommodations?” and “How can I adjust JupyterLab to work better for me with assistive technology?”
```
Comment (trallard) [inline docs/funding/google-season-of-docs-2023.md:33]:
```suggestion
Through this project, we aim to create documentation primarily targeted at anyone with a temporary, situational, or permanent disability that wants or needs to use JupyterLab. We want to provide the JupyterLab community with a comprehensive guide about the accessibility features present in the application. For example, developing documentation that answers questions such as “What can and can’t I do in JupyterLab with accessibility accommodations?” and “How can I adjust JupyterLab to work better for me with assistive technology?”
```

File: docs/funding/google-season-of-docs-2023.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+- An authoring guide focused on best practices to generate accessible notebooks.
+
+Work that is out of the scope of this project includes:
+
+- Creation of developer-focused documentation
+- Tutorials and workshops to present in conferences
+- Live streams and social media engagement
+
+The developer that will be mentoring this project has previously mentored successful GSoD in another project last year and has been involved in the accessibility remediation work for JupyterLab. We don’t have a technical writer for this project, but we intend to start the  application and selection process once we receive the acceptance by Google Season of Docs.
```
Comment (trallard) [inline docs/funding/google-season-of-docs-2023.md:56]: Do we need to name the mentor?

File: docs/funding/google-season-of-docs-2023.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+We estimate that this project will be completed by one technical writer working full-time in a span of six months.
+
+## Measuring your project’s success
+
+<!---
+How will you know that your new documentation has helped solve your problem? What metrics will you use, and how will you track them?
+-->
+
+Even though JupyterLab users vary in their experience, knowledge and confidence of the application, there’s an overall unfamiliarity about the accessibility features available within it. This is mostly a result of the lack of a reliable and official documentation that can guide users through different situations like adjusting theming, keyboard support and assistive technologies support. There have been numerous questions raised about accessibility features in the issue tracker of the project and in general, the community wants to move to a more inclusive access to the information given that JupyterLab is one of the most common tools used in education, research and scientific development.
```
Comment (trallard) [inline docs/funding/google-season-of-docs-2023.md:66]:
```suggestion
Even though JupyterLab users vary in their experience, knowledge and confidence of the application, there’s an overall unfamiliarity about the accessibility features available within it. This is primarily a result of the lack of reliable and official documentation that can guide users through different situations like adjusting theming, keyboard and assistive technologies support. There have been numerous questions raised about accessibility features in the issue tracker of the project, and in general, the community wants to move to more inclusive access to the information given that JupyterLab is one of the most common tools used in education, research and scientific development.
```

File: docs/funding/google-season-of-docs-2023.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+We will track three metrics to measure this documentation project's success. The first is the number of accessibility features documented; the second is the number of pages added regarding the audits for keyboard navigation and 400% zoom.
```
Comment (trallard) [inline docs/funding/google-season-of-docs-2023.md:68]: This mentions three metrics - though I can only see two

1. No features documented
2. Zoom and keyboard pages

Seems this needs adjusting

```suggestion
We will track two metrics to measure this documentation project's success. The first is the number of accessibility features documented; the second is the number of pages added regarding the audits for keyboard navigation and 400% zoom.
```

File: docs/funding/google-season-of-docs-2023.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+Design work to create branding, logos, templates, or other design assets for your documentation site
+Minimal amounts (<US$200) for project swag (t-shirts or stickers for your participants). If you use the Season of Docs logo, it must be accompanied by your project or organization logo or name. Your swag may not use the name Google.
+Minimal stipends for volunteers who take on considerable mentorship or guidance roles in the project (we recommend no more than $500 per volunteer, please)
+Downstream donations to other open source projects should be no more than 10% of your budget total.
+Include other budget items as needed, along with justification for the amount sought. Expense justifications should highlight how the expenditure will contribute to the success of the project as a whole.
+-->
+
+| Budget item | Amount | Running Total | Notes/Justifications |
+| :--:    |      :---:   | :--:    |      :---:   |
+| Technical writer that will update, test and publish new accessibility documentation |$10000 |$10000 |The mentors’ time will be provided as in-kind donation by Quansight Labs so the budget will be used in full for the technical writer’s compensation. |
```
Comment (trallard) [inline docs/funding/google-season-of-docs-2023.md:107]: Maybe add a link to Quansight Labs site

**Review by trallard — CHANGES_REQUESTED** [review]
I added a number of questions and suggestions, but this seems in a good place. 

Echoing @isabela-pf it would be best to add some more links to projects for whoever reviews this (esp. if not super familiar with the project(

File: docs/funding/google-season-of-docs-2023.md (hunk 5 above)
Comment (steff456) [reply docs/funding/google-season-of-docs-2023.md:56]: Last year we didn't named anyone in the proposal.

**Comment by steff456** [comment]
Sorry everyone, I'm not sure how I mixed the dates up and thought the deadline was the 31st and not the 24th. I'm closing this PR for now, will re-open when we have a new grant to submit this project to.

**Pull request closed without merging**